addon/webextension/background/startBackground.js

```diff
@@ -0,0 +1,94 @@
+/* globals browser, main, communication */
+/* This file handles browser.browser.onClicked, and loads the rest of the background
+   page in response to that event. */
+
+this.startBackground = (function() {
+  const backgroundScripts = [
+    "log.js",
+    "makeUuid.js",
+    "catcher.js",
+    "background/selectorLoader.js",
+    "background/communication.js",
+    "background/auth.js",
+    "background/senderror.js",
+    "build/raven.js",
+    "build/shot.js",
+    "background/analytics.js",
+    "background/deviceInfo.js",
+    "background/takeshot.js",
+    "background/main.js"
+  ];
+
+  browser.browserAction.onClicked.addListener((tab) => {
+    loadIfNecessary().then(() => {
+      main.onClicked(tab);
+    }).catch((error) => {
+      console.error("Error loading Screenshots:", error);
```

Comment:
Hmm. As of right now, if `loadIfNecessary` fails, the user will see "Error loading Screenshots" in the console, and we won't be able to send the failure to Sentry (since the raven.js script may or may not have loaded).

Is there any way we could inline a Sentry ping inside this file, so that we can detect worst-case failure scenarios while gradually rolling out a new release? If we take that approach, it would be nice to add a `catch` inside `loadIfNecessary`, to avoid copying the error handling code around.

Also: if loading scripts fails, does it even make sense to keep the Screenshots toolbar button and context menu item available? Should we just tear down the entire feature?